game/game.py

```diff
@@ -74,6 +74,7 @@ def __init__(self) -> None:
         )
         self.current_player = self.player_1
         self.game_is_active = True
+        self.remaining_moves = 200
```

Comment:
What's the point of `remaining_moves` being `200`? Is a "write only" variable. Is the check going to be implemented on a different task?

Reply:
The game only going to have 200 turns, there is gonna be another module probably called `manage.py` that controls all the games on the platform.  After every player action, it's going to check if the current game it's over or continues with the next player.